checker/check-response-property-one-of-updated.go

```diff
@@ -0,0 +1,111 @@
+package checker
+
+import (
+	"fmt"
+
+	"github.com/tufin/oasdiff/diff"
+)
+
+func ResponsePropertyOneOfUpdated(diffReport *diff.Diff, operationsSources *diff.OperationsSourcesMap, config Config) Changes {
+	result := make(Changes, 0)
+	if diffReport.PathsDiff == nil {
+		return result
+	}
+
+	for path, pathItem := range diffReport.PathsDiff.Modified {
+		if pathItem.OperationsDiff == nil {
+			continue
+		}
+
+		for operation, operationItem := range pathItem.OperationsDiff.Modified {
+			if operationItem.ResponsesDiff == nil || operationItem.ResponsesDiff.Modified == nil {
+				continue
+			}
+			source := (*operationsSources)[operationItem.Revision]
+
+			for responseStatus, responsesDiff := range operationItem.ResponsesDiff.Modified {
+				if responsesDiff.ContentDiff == nil || responsesDiff.ContentDiff.MediaTypeModified == nil {
+					continue
+				}
+
+				modifiedMediaTypes := responsesDiff.ContentDiff.MediaTypeModified
+				for _, mediaTypeDiff := range modifiedMediaTypes {
+					if mediaTypeDiff.SchemaDiff == nil {
+						continue
+					}
+
+					if mediaTypeDiff.SchemaDiff.OneOfDiff != nil && mediaTypeDiff.SchemaDiff.OneOfDiff.Added > 0 {
+						result = append(result, ApiChange{
+							Id:    "response-body-one-of-added",
+							Level: INFO,
```

Comment:
Based on the same logic as request-property-one-of-removed is breaking, shouldn't this also be breaking?

Reply:
I think in general responses we don't treat additions as breaking changes (see ResponsePropertyEnumValueAddedCheck, ResponseRequiredPropertyUpdatedCheck), while the same scenarios are considered breaking changes for request body.
Do you have a different opinion for `oneOf`? (no strong opinion on my side)

Reply:
My approach is that request and response should be treated alike.
I'll open a discussion to collect more feedback from users and then we can make a decision.